powershell/gulpfile.js

```diff
@@ -2,12 +2,21 @@ var fs = require('fs');
 var path = require('path');
```

Comment:
Gulpfile changes. I'm pretty sure I did this correctly, but wanted to make sure these new dependencies are OK.